src/pages/index.html

```diff
@@ -1,68 +1,68 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>[REDACTED]</title>
-        <!--Default stylesheet (required)-->
-        <link rel="stylesheet" href="../components/styles/stylesheet.css">
-        <!--stylesheet for scp page-->
-        <link rel="stylesheet" href="../components/styles/page-content.css">
-        <!-- Simple trick to make things work so you can use import/exports of typescript, 
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>[REDACTED]</title>
+    <!--Default stylesheet (required)-->
+    <link rel="stylesheet" href="../components/styles/stylesheet.css" />
+    <!--stylesheet for scp page-->
+    <link rel="stylesheet" href="../components/styles/page-content.css" />
+    <!-- Simple trick to make things work so you can use import/exports of typescript, 
```

Comment:
Try not to use void tags. I configured prettier to automatically add closing tags to those that need it.